schemas/2014-08-01-preview/Microsoft.Scheduler.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "http://schema.management.azure.com/schemas/2014-08-01/Microsoft.Scheduler.json",
+  "id": "http://schema.management.azure.com/schemas/2014-08-01-preview/Microsoft.Scheduler.json",
```

Comment:
Please update references in 2015-01-01\deploymentTemplate.json and 2014-04-01-preview\deploymentTemplate.json